src/components/NavBar.js

```diff
@@ -6,8 +6,8 @@ const NavBar = () => (
     <nav className="nav-bar">
       <p>Bookstore CMS</p>
       <ul className="links-nav">
-        <li><Link to="/"> Books </Link></li>
-        <li><Link to="/Categories"> Categories </Link></li>
+        <li><Link to="/" className="nav-link"> Books </Link></li>
+        <li><Link to="/Categories" className="nav-link"> Categories </Link></li>
```

Comment:
- [ ] Kindly make sure your design same as the original design.
- [ ] All texts, buttons, etc so small on this page.
- [ ] Kindly add a user icon on the right side
- [ ] Kindly follow the layout of the original design(space between elements)

![image](https://user-images.githubusercontent.com/38635158/188289844-b06acd53-6168-4665-bc38-6e6c2acade25.png)
![image](https://user-images.githubusercontent.com/38635158/188289877-9168abde-3beb-42c9-a14b-bc1a564c9653.png)